### DISCUSSION TITLE
トラックボールが反応しなくなった

### DISCUSSION THREAD

**Discussion by hashimoto76** [Yowkees/keyball#270] [General]
先日まで正常に動いていた、トラックボールによるカーソル操作が行えなくなりました。
ほかのキーは問題なく動きますが、トラックボールの操作によるカーソル移動、レイヤ1にある右クリック及び左クリック、レイヤ3でのスクロール動作など、マウスの動作に関する操作のみが無効になっているようです。
トラックボールを動かすとpromicroの上での数字(移動量？)は問題なく増減しているため、センサーの故障やボール付近のジャンパ不良などではなさそうです。
左右のpromicroの入れ替えや、remapの上書きなど様々なことを試しましたが改善しませんでした。
どのような解決方法が考えられますでしょうか。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

OLEDにはトラックボールの移動量が出ているということですので、ハード以外に問題がありそうです。
カーソル操作が別デバイスに乗っ取られてるような気がします。別のPCに接続すると正常なのかご確認いただければ幸いです。

お手数ですがご対応お願いいたします。

**Reply by hashimoto76** [reply]
keyballと同時に接続していた無線マウスを外すとちゃんと反応するようになりました。
(以前は同時に使えてはいたのですが...)
ご対応ありがとうございます。
